zeebe/qa/util/src/main/java/io/camunda/zeebe/qa/util/cluster/TestStandaloneBroker.java

```diff
@@ -63,9 +63,12 @@ public int mappedPort(final TestZeebePort port) {
     };
   }
 
-  @Override
   protected SpringApplicationBuilder createSpringBuilder() {
-    return super.createSpringBuilder().profiles(Profile.BROKER.getId());
+    // because @ConditionalOnRestGatewayEnabled relies on the zeebe.broker.gateway.enable property,
+    // we need to hook in at the last minute and set the property as it won't resolve from the
+    // config bean
```

Comment:
:question: Should we add this comment as a Javadoc somewhere? It might be useful for the behavior to be clearer, in case someone decides to use this class together with a `TestStandaloneBroker`.